.eslintrc.json

```diff
@@ -59,28 +59,24 @@
       {
         "paths": [
           {
-            "name": "lodash",
-            "message": "Please import from 'lodash/module' directly to support tree-shaking."
-          },
```

Comment:
intentional removal?

Reply:
We no longer use lodash, so I took it out as tech debt.